### PR TITLE
Add sub-apps for compliance

### PR DESCRIPTION
**Pull request by dLobatog**
We were asked to split the app into Reports, Policies and Systems. However Policies will not be available in QA and prod for a while, how should we work around that?

This PR introduces "policies" - https://github.com/RedHatInsights/compliance-frontend/pull/197

### CODE REVIEW

**Comment by ryelo** [comment]
@dLobatog when you say
> However Policies will not be available in QA and prod for a while, how should we work around that?

Do you mean policies won't be in qa-stable and prod-stable?

**Review by dLobatog — COMMENTED** [review]
@ryelo That's right - it will sit on CI only for a while, as we want to add a few things to it before it can be deployed to other envs. It won't be useful at all in prod yet (changes to the client need to be made for that), so we don't want it there yet in order to avoid chances of breaking prod.

**Comment by ryelo** [comment]
That's fine. When we push this out to the stable environments, we can leave the policies out.

File: main.yml [hunk 1]
```diff
@@ -108,6 +108,17 @@ compliance:
   frontend:
     paths:
       - /rhel/compliance
+    sub_apps:
+      - id: ''
+        title: Reports
+        default: true
+        reload: compliance
```
Comment (kruai) [inline main.yml]: Has this been tested locally? I haven't seen a `reload` defined for a default app before. If it was tested and working, I'll go ahead and approve this

Comment (dLobatog) [reply main.yml]: Yeah, it works well - I removed it though and it works the same, to be fair I just took the `drift` app as an example and thought it was necessary.

**Review by dLobatog — COMMENTED** [review]
It seems to work well as you can see in:

Screenshot from 2020-01-20 09-53-55

However if I click around the vertical nav in beta, for some reason it seems to point to the stable URL. Any idea what's going on there?

Peek 2020-01-20 09-52

Navigation in compliance-frontend is changed in beta through https://github.com/RedHatInsights/compliance-frontend/pull/287

**Review by kruai — APPROVED** [review]

**Comment by ryelo** [comment]
@dLobatog It's probably your app loading the stable version of chrome versus the beta version of chrome

**Review by akofink — APPROVED** [review]
Looks good. :+1:

**Comment by kruai** [comment]
@ryelo If you think this looks good as well, I'll go ahead and merge!

File: main.yml [hunk 2]
```diff
@@ -104,10 +104,19 @@ compliance:
   api:
     versions:
       - v1
+    isBeta: true
   deployment_repo: https://github.com/RedHatInsights/compliance-frontend-build
   frontend:
     paths:
       - /rhel/compliance
+    sub_apps:
+      - id:
```
Comment (kruai) [inline main.yml]: This should be `- id: ''` if you want the ID to be blank.

**Review by kruai — CHANGES_REQUESTED** [review]
Actually, there's one small change that needs to be made before merge.

**Review by ryelo — APPROVED** [review]

**Review by kruai — APPROVED** [review]

**Pull request merged**